package.json

```diff
@@ -2,7 +2,7 @@
   "name": "balena-cli",
   "version": "11.9.3",
   "description": "The official balena CLI tool",
-  "main": "./build/actions/index.js",
+  "main": "./build/app.js",
```

Comment:
I don't think this change has a practical effect, but it better reflects the entrypoint.